UIkitDZ/VC/LeftViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  ViewController.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 30.06.2021.
+//
+
+import UIKit
+
+final class LeftViewController: UIViewController {
+    
+    private var demonstrationLabel = UILabel()
+    private var dollyText = String()
+    private var colorPicker = UIPickerView()
+    private var numberOfLinesPicker = UIPickerView()
+    private var segmentedControll = UISegmentedControl()
+    private var sizeFontSlider = UISlider()
+    private var shadowSlider = UISlider()
+    private var typeFontArr = ["Italic", "Bold", "Regular"]
+    private var colorArr = [UIColor.blue, UIColor.red, UIColor.orange, UIColor.brown, UIColor.green]
+    private var colorNameArr = ["blue", "red", "orange", "brown", "green"]
+    private var numberOfLinesArr = Array(0...10).map { String($0) }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(false)
+        shadowSlider.value = 50
+        sizeFontSlider.value = 50
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        demonstrationLabel = createLabel(textExample: dollyText, x: 50, y: 150)
+        segmentedControll = createSegment(x: 115, y: 110)
+        segmentedControll.addTarget(self, action: #selector(makeChanges), for: .valueChanged)
+        sizeFontSlider.frame = CGRect(x: 60, y: 520, width: 280, height: 20)
+        sizeFontSlider.minimumValue = 0
+        sizeFontSlider.maximumValue = 100
+        sizeFontSlider.minimumTrackTintColor = UIColor.red
+        sizeFontSlider.maximumTrackTintColor = UIColor.blue
+        sizeFontSlider.addTarget(self, action: #selector(changeFontSize), for: .valueChanged)
+        view.addSubview(sizeFontSlider)
+        shadowSlider.frame = CGRect(x: 60, y: 580, width: 280, height: 20)
+        shadowSlider.minimumValue = 0
+        shadowSlider.value = 25
+        shadowSlider.minimumValueImage = UIImage(systemName: "minus.circle")
+        shadowSlider.maximumValueImage = UIImage(systemName: "plus.circle")
+        shadowSlider.maximumValue = 100
+        shadowSlider.addTarget(self, action: #selector(changeShadowDistance), for: .valueChanged)
+        view.addSubview(shadowSlider)
+        var logoItem = UIImageView(frame: CGRect(x: 0, y: 0, width: 20, height: 15))
+        colorPicker = createPicker(x: 40, y: 400)
+        colorPicker.dataSource = self
+        colorPicker.delegate = self
+        
+        numberOfLinesPicker = createPicker(x: 200, y: 400)
+        numberOfLinesPicker.dataSource = self
+        numberOfLinesPicker.delegate = self
+        logoItem.contentMode = .scaleAspectFit
+        logoItem = UIImageView(image: UIImage(systemName: "moon.stars"))
+        navigationItem.titleView = logoItem
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addTextIntoLabel))
+    }
+    
+    // MARK: -делаем алерт, добавляем текст в лэйбл
```

Comment:
```suggestion
    // MARK: -делаем алерт, добавляем текст в лэйбл
```
Марки не пишем на русском пишем по гайду // MARK: - Actions